packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -943,7 +946,15 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
      * @deprecated - Use summarize to get summary of the container runtime.
      */
     public async snapshot(): Promise<ITree> {
-        const root: ITree = { entries: await this.dataStores.snapshot() };
+        const root: ITree = { entries: [] };
+
+        root.entries.push(new TreeTreeEntry(
+            channelsTreeName,
+            { entries: await this.dataStores.snapshot() },
+        ));
+
+        const metadata: IContainerRuntimeMetadata = { snapshotFormatVersion: 1 };
+        root.entries.push(new BlobTreeEntry(metadataBlobName, JSON.stringify(metadata)));
```

Comment:
Not really sure if I needed to change `snapshot()` or not, since it is deprecated and intended to be removed.

Reply:
The snapshot tests use this. IMO, we should update it to get coverage on this new change.